tests/js/JSONFormatBearTest.py

```diff
@@ -35,6 +35,9 @@
 }"""
 
 
+max_line_length_file = '{\n    "string": "' + 'a' * 100 + '"\n}'
```

Comment:
beautifully correctly ugly.

but you are only testing very large, 100 or more.

correct testing of lenth '79' requires test data with 79 and 80 chars , because that is the trigger range - that is how we know your algorithm activates at exactly the right number of characters, and not before, and not after.

Reply:
Okay, I'll modify the char size. Though the four spaces before string are proving to be ugly. Is there any workaround?